cocos/tween/actions/action-interval.ts

```diff
@@ -493,17 +497,20 @@ export class Repeat extends ActionInterval {
     private _actionInstant = false;
     private _innerAction: FiniteTimeAction | null = null;
 
-    constructor (action?: any, times?: any) {
+    constructor (action: FiniteTimeAction | null = null, times: number = 0) {
         super();
-        times !== undefined && this.initWithAction(action as FiniteTimeAction, times as number);
+        this.initWithAction(action, times);
     }
 
     /*
      * @param {FiniteTimeAction} action
      * @param {Number} times
      * @return {Boolean}
      */
-    initWithAction (action: FiniteTimeAction, times: number): boolean {
+    initWithAction (action: FiniteTimeAction | null, times: number): boolean {
+        if (!action || times === undefined) {
```

Comment:
Check times here just to keep compatibility?

Reply:
I will revert it to 
```ts
initWithAction (action?: FiniteTimeAction, times?: number): boolean {
```